src/components/layout/footer/main-links.js

```diff
@@ -152,23 +152,13 @@ const MainLinksSection = ({ is_ppc, is_ppc_redirect }) => {
                             <Title>{localize('PARTNER')}</Title>
                         </LinkWrapper>
                         <LinkWrapper>
-                            <Link to="/partners/affiliate-ib/">
-                                {localize('Affiliates and IBs')}
-                            </Link>
+                            <Link to="/partners/affiliate-ib/">{localize('Affiliates & IBs')}</Link>
                         </LinkWrapper>
                         <LinkWrapper>
                             <Link to="/partners/payment-agent/">{localize('Payment agents')}</Link>
                         </LinkWrapper>
                         <LinkWrapper>
-                            <Link
-                                to=""
-                                type="api"
-                                target="_blank"
-                                external="true"
-                                rel="noopener noreferrer"
-                            >
-                                {localize('API')}
-                            </Link>
+                            <Link to="https://api.deriv.com/">{localize('API')}</Link>
```

Comment:
Won't hardcoding the API URL always redirect to live version? this will be an issue for example in the staging version, it will redirect to the live version instead of the staging version.

Reply:
This is an external site so I don't see an issue here. But what we can optimise for is the domain itself like `.me` and `.be` so we should instead use `deriv_api_url` here that will handle that.